adapters/clients/remote_index.go

```diff
@@ -882,3 +881,48 @@ func (c *RemoteIndex) FindObject(ctx context.Context, hostName, indexName,
 	}
 	return obj, c.retry(ctx, 9, try)
 }
+
+func (c *RemoteIndex) OverwriteObjects(ctx context.Context,
+	host, index, shard string, objects []*objects.VObject,
+) ([]*objects.VObject, error) {
```

Comment:
I think the return type should be `[]replica.RepairResponse` (see `putOverwriteObjects() adapters/handlers/rest/clusterapi/indices.go`)

Reply:
Nice catch, fixed